code/datums/jobs/job/survivor.dm

```diff
@@ -252,10 +396,74 @@ Good luck, but do not expect to survive."})
 	name = "Salesman Survivor"
 	jobtype = /datum/job/survivor/salesman
 
-	w_uniform = /obj/item/clothing/under/liaison_suit
-	wear_suit = /obj/item/clothing/suit/armor/vest
+	w_uniform = /obj/item/clothing/under/lawyer/purpsuit
+	wear_suit = /obj/item/clothing/suit/storage/lawyer/purpjacket
 	shoes = /obj/item/clothing/shoes/black
 	back = /obj/item/storage/backpack/satchel
+	belt = /obj/item/storage/bag/cash
+	mask = /obj/item/clothing/mask/cigarette/pipe/bonepipe
+	glasses = /obj/item/clothing/glasses/sunglasses/aviator
+	ears = /obj/item/radio/survivor
+
+/datum/outfit/job/survivor/salesman/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/holdout, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/lighter/zippo, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/crowbar, SLOT_IN_BACKPACK)
+
+	H.equip_to_slot_or_del(new /obj/item/spacecash/c500, SLOT_IN_BELT)
```

Comment:
Jeez, please just do a for loop

Reply:
thinking back, its a stupid meme so i'm gonna get rid of the cash altogether